low-level/files/opt/seagate/sspl/setup/sspl_setup.py

```diff
@@ -501,10 +505,63 @@ def validate(self):
 
     def process(self):
         try:
+            args = ' '.join(self._args.args)
+            Conf.load(SSPL_CONFIG_INDEX, sspl_config_path)
+            sspl_log_file_path = Utility.get_config_value(
+                    SSPL_CONFIG_INDEX, "SYSTEM_INFORMATION>sspl_log_file_path")
+            iem_log_file_path = Utility.get_config_value(
+                    SSPL_CONFIG_INDEX, "IEMSENSOR>log_file_path")
             if os.path.exists(file_store_config_path):
                 os.remove(file_store_config_path)
             shutil.copyfile("%s/conf/sspl.conf.%s.yaml" % (
                 SSPL_BASE_DIR, self.product), file_store_config_path)
+
+            # --pre-factory will cleanup config and undo everything whatever
```

Comment:
Maintain in sspl_cleanup itself, under a separate function, which process() would call.

Reply:
we don't have sspl_cleanup file. all cleanup operation we have maintained here in CleanupCmd class.

Reply:
thats not right, guess whoever introduced that missed it. lets create separate sspl_cleanup functionality and maintain relevant code there.

Also avoid 'we' in comment on line 520, keep it as
> \# --pre-factory will cleanup config and undo everything whatever was done in post-install Mini-Provisioner Interface.